hieradata/vagrant/common.yaml

```diff
@@ -69,7 +69,7 @@ profile::jenkinscontroller::jcasc:
         credentialsId: "ppc64le-ed25519-2021-12-03"
         launchTimeoutSeconds: 713
         retryWaitTime: 175
-        hostKey: "ssh-ed25519 whatever"
+        hostKey: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCYwX3Fu4fWEb9hPlan3FsiUsZoCsMD7CMqFNT/+Uh11HhvKU5ZFiF3sGNI7FrhNNbqBFd0HnS1t3zIkP3FFlKToSTrkcXPUyw+svFf5YbPbDxQUNE0kclTUsERzC3GNB5eXUlyNxCiGksqGtinXgknF2Z5cOO8osODP7ddRc6L3H4gvDi0/smz9QukZB2N0FqBJ3EZGbv0X9V3iwRu6Cu9lhcl/ue5fuIjKAgGzGQgWgCEg0k9xkK0OmxyJalI0eiqBRbdES/bXkkxp+4TQNOsxN/ZrZqxtlN7o9Fq9y+dp7xQFEoivSAjn6WQ6izjkMGKon7rcFJ4t4g6FJoQYv2P"
```

Comment:
Side note for later reader: this line change fix the vagrant provisioning (an invalid RSA key fails the provisioning)